login/login.c

```diff
@@ -445,6 +437,19 @@ int login_authenticate(glome_login_config_t* config, pam_handle_t* pamh,
     return failure(EXITCODE_INVALID_INPUT_SIZE, error_tag, "authcode-length");
   }
 
+  size_t min_len = MIN_ENCODED_AUTHCODE_LEN;
+  if (config->min_authcode_len > min_len) {
```

Comment:
I think we should error out if a min authcode length larger than the tag length is being requested. (Because you'd never be able to satisfy that.)

Reply:
Added a check for that case.